src/particles/deposition/PlasmaDepositCurrentInner.H

```diff
@@ -95,9 +95,16 @@ void doDepositionShapeN (const bool use_laser,
 {
     using namespace amrex::literals;
 
-    static constexpr int outer_depos_order_1 = outer_depos_loop ? (depos_order_xy + 1) : 1;
-    static constexpr int outer_depos_order_1_sq = outer_depos_order_1 * outer_depos_order_1;
-    static constexpr int inner_depos_order = outer_depos_loop ? 0 : depos_order_xy;
+    static constexpr bool outer_depos_loop_x = outer_depos_loop;
+    static constexpr bool outer_depos_loop_y = false;
```

Comment:
Can we remove this option, if it is not meant for being used? That should simplify the code quite a bit (also helping with outer_depos_order_xy_1_sq etc.), and we can refer to this commit if it has a chance of being useful on a future platform.
```suggestion
    // Using 1 thread per particle and per deposited cell is only done in the fast (x) direction.
    // This can also be applied in the y direction, but so far does not show significant gain.
    static constexpr bool outer_depos_loop_x = outer_depos_loop;
```